plugins/plugin-site/src/components/Plugin.jsx

```diff
@@ -36,7 +36,7 @@ function Plugin({plugin: {name, title, stats, labels, excerpt, developers, build
             </div>
             <div className="Plugin--InstallsContainer">
                 {'Installs:  '}
-                {stats.currentInstalls}
+                {stats.currentInstallPercentage ? formatPercentage(stats.currentInstallPercentage) : '?'}
```

Comment:
When there are no usage stats, it's 0 (or 0%).

We could mention that for new plugins, it's expected, but there are a few plugins hovering around 1-2 installs, and for them, 0 means 0, not "too new".

Reply:
With no usage stats we already store 0. This is just to avoid failing when the site expects new format and Algolia has old one (happens on PR preview site + maybe temporarily after this is merged and Algolia is updating...).